wiki/Tournaments/Template/en.md

```diff
@@ -0,0 +1,149 @@
+---
+tags:
+- {ABBR}
+---
+
+<!--
+Find and replace those tags listed below with proper information, then delete this comment.
+{TOURNAMENT_NAME} = tournament name
+{MODE} = game mode
+{ABBR} = abbreviation of the tournament name followed by its iteration/year occurence i.e. MWC 4K 2018
+{ORGANISER} = organiser of the tournament
+{NTH} = installment ordinal number
+{DRAW_TIME} = Live Drawings (Livestream) time for the tournament
+{TOPIC_NUM} = forum thread number
+{STATS_LINK} = link to overall statistics sheet (it should be a Google Drive link)
+You will still need to fill other items such as tables, etc.
+Use .png as a default format for images, since .jpg can't handle transparency.
+Alongside this comment, remember to delete all comments across the article after you finish.
+-->
+
+# {TOURNAMENT_NAME}
+
+![{ABBR} Logo](img/logo.png)
+
+The **{TOURNAMENT_NAME}** (***{ABBR}***) is a country-based {MODE} tournament hosted by the {ORGANISER}. It is the {NTH} installment of the {TOURNAMENT_NAME}.
```

Comment:
`{NTH} installment of the {TOURNAMENT_NAME}` -> `{NTH} installment of the {TOURNAMENT_SERIES}`

`SERIES` is the one without the iteration suffix.